src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -185,7 +200,7 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
     }
 
     refresh() {
-        this.getResults();
+        this.getResults().subscribe();
```

Comment:
If there is no logic inside subscribe, I think you don't even have to subscribe

Reply:
It is needed, without the subscribe the Observable won't be fired.